src/ui/server/controllers/insurance/business/contact/index.ts

```diff
@@ -74,6 +77,7 @@ const get = (req: Request, res: Response) => {
       }),
       userName: getUserNameFromSession(req.session.user),
       application: mapApplicationToFormFields(application),
+      applicationAnswers: getFromSessionOrApplication(application, BUSINESS, BUSINESS_CONTACT_DETAIL, req.session.user),
```

Comment:
What do you think about calling this `submittedValues`, same as other places? Also `getFromSessionOrApplication` doesn't feel descriptive enough to me, how about `getSubmittedValues()`?

Reply:
I think `submittedValues` makes sense even if that name is primarily used in post's - `applicationAnswers`doesn't feel right because it implies it will be from an application when it's not. I can't think of a more appropriate name, what do you think?